draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -890,20 +907,29 @@ and header fields, and a response payload.
 
 This content type consists of the concatenation of the following items:
 
-1. The ASCII characters "sxg1-b1" followed by a 0 byte, to serve as a file
+1. The ASCII characters "sxg1-b2" followed by a 0 byte, to serve as a file
    signature. This is redundant with the MIME type, and recipients that receive
    both MUST check that they match and stop parsing if they don't.
 
    Note: As this is a snapshot of a draft of
    {{?I-D.yasskin-http-origin-signed-responses}}, it uses a distinct file
    signature.
-1. 3 bytes storing a big-endian integer `sigLength`. If this is larger
-   than 16kB, parsing MUST fail.
+1. 2 bytes storing a big-endian integer `fallbackUrlLength`.
```

Comment:
(Given 65536 is longer than several existing browsers can support (AFAICT), perhaps the fetch spec should say how browsers should respond if the fallbackUrl is too long? Or specify a minimum length they must support? Or is this already covered by some other part of Fetch?)

Reply:
I'm not sure. Would you mind filing a separate issue?

Reply:
OK, filed #297.